nixos/modules/services/networking/firewall.nix

```diff
@@ -493,6 +510,51 @@ in
         '';
     };
 
+    networking.firewall.extraAllowed = mkOption {
+      type = types.listOf (types.submodule { options = {
+           sourceAddress = mkOption {
+                type = types.nullOr types.string;
+                default = null;
+                example = "198.51.100.0";
+                description = "Source of the packets. According to the iptables manual, DNS host names might be used here, but their use is discouraged (in particular because this delegates the security rule to DNS servers). DNS host names are resolved only when the rule is loaded.";
```

Comment:
Done. OTOH, some emacs users (including myself) are also used to mode and formatting tools that apply style guidelines automatically…